packages/core/android/src/main/java/com/datadog/reactnative/DatadogWrapper.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
+
+import android.content.Context
+import com.datadog.android.core.configuration.Configuration
+import com.datadog.android.core.configuration.Credentials
+import com.datadog.android.privacy.TrackingConsent
+import com.datadog.android.rum.RumMonitor
+
+internal interface DatadogWrapper {
```

Comment:
Why do you have 2 ? `DatadogWrapper` and `DatadogSdkWrapper` ?

Reply:
I think it's for testing, `DatadogWrapper` is the interface and `DatadogSdkWrapper` the implementation. I copied it over from the bridge, maybe it'll make more sense once I bring the tests

Reply:
I see, my bad, I didn't see that.